tests/tests_unit/test_api/test_functions.py

```diff
@@ -652,6 +694,46 @@ def test_functions_status_endpoint(self, mock_functions_status_response, cognite
         assert isinstance(res, FunctionsStatus)
         assert mock_functions_status_response.calls[1].response.json() == res.dump(camel_case=True)
 
+    def test_last_called_field_on_created_function(self):
```

Comment:
These tests should live under test_data_classes instead. But note that we do have automatically generated tests for dumping/loading of CogniteObjects, so these may be redundant?